src/l10n.json

```diff
@@ -152,6 +152,8 @@
     "registration.confirmYourEmail": "Confirm Your Email",
     "registration.confirmYourEmailDescription": "If you haven't already, please click the link in the confirmation email sent to:",
     "registration.createUsername": "Create a username",
+    "registration.emailStepTitle": "What's your email?",
```

Comment:
This is not good, because for children under 16 they need parent's email. In that case this should be `Ask your parents or guardians to fill it`

Reply:
Thanks for the feedback! After careful consideration and discussion with our legal team we have decided to use this language for the email field.